cub/cub/thread/thread_scan.cuh

```diff
@@ -164,6 +164,87 @@ ThreadScanExclusive(T (&input)[LENGTH], T (&output)[LENGTH], ScanOp scan_op, T p
   return ThreadScanExclusive<LENGTH>((T*) input, (T*) output, scan_op, prefix, apply_prefix);
 }
 
+/**
+ * @brief Perform a sequential exclusive prefix scan over the first @p valid_items elements
+ *        of the statically-sized @p input array, seeded with the specified @p prefix.
+ *        The aggregate is returned.
```

Comment:
Strong suggestion: please prefer `//!` style documentation comments
```suggestion
/**
//! @brief Perform a sequential exclusive prefix scan over the first @p valid_items elements
//!       of the statically-sized @p input array, seeded with the specified @p prefix.
//!       The aggregate is returned.
```
See also (the draft of) our [coding guidelines](https://github.com/NVIDIA/cccl/wiki/Cpp-Coding-Guidelines). Btw, since you are onboarding still, let me know if you discover any new guidelines we should add. You can make suggestions at #2635.

Reply:
I just kept with what was already in this file. Given that this is for now internal and wont land on the documentation website, I thought it would be better to change this once (if ever) this becomes a public API.